packages/flutter/lib/src/rendering/flex.dart

```diff
@@ -470,6 +470,9 @@ class RenderFlex extends RenderBox with ContainerRenderObjectMixin<RenderBox, Fl
 
   // Set during layout if overflow occurred on the main axis.
   double _overflow;
+  // Check whether any meaningful overflow is present. Values below an epsilon
+  // are treated as not overflowing.
+  bool get _hasOverflow => _overflow > 1e-9;
```

Comment:
We have a epsilon defined in https://github.com/flutter/flutter/blob/a8504405a8dcad08653fc01c2ccd1fe19d0970d7/packages/flutter/lib/src/rendering/sliver.dart#L690

I wonder we can reuse this or define this constant somewhere else more meaningful

Reply:
Moving it makes sense.  I'm not sure where to though.  Foundation?  \

I could also just `import sliver.dart show precisionErrorTolerance`